app/middleware/request_validations.py

```diff
@@ -0,0 +1,70 @@
+"""This module contains Middleware class"""
+
+
+class Middleware:
+    """
+    This class handles code submission validation for the application.
+    The Middleware class validates incoming code submissions and ensures
+    they meet the necessary criteria.
+    """
+
+    def __init__(self, data_dict):
```

Comment:
Me falta configurar algo del pre-commit pero pon tipado de los parametros (asi no tienes que colocar data_dict sino, data: dict

Reply:
resolve in https://github.com/MinervaLMS/judge-api/commit/818f80c227114367814f43b7fdb5435c8b8618c6